tests/foreman/cli/test_variables.py

```diff
@@ -1151,7 +1170,6 @@ def test_positive_enable_merge_overrides_default_flags(self):
             smart_variable['override-values']['merge-default-value'], True)
 
     @run_only_on('sat')
-    @skip_if_bug_open('bugzilla', 1375652)
```

Comment:
Bug is still in ON_QA state, so please whether verify that bug and put VERIFIED state or do not remove that line